pkg/performanceprofile/cmd/render/render.go

```diff
@@ -213,6 +213,11 @@ func render(inputDir, outputDir string) error {
 			componentObj.SetOwnerReferences(or)
 		}
 
+		// TODO: Resolve why kubelet ownership breaks bootstrap
+		components.KubeletConfig.SetOwnerReferences(nil)
```

Comment:
generalizing here: https://github.com/openshift/cluster-node-tuning-operator/pull/907

Reply:
Nice! Should we keep this here for now and remove it in your PR?

Reply:
Either work. If we keep the fix in this PR, I'd avoid setting the OwnerReference in the first place rather than adding (lines 203-214) and then remove it right afterwards.
On top of that, I'm quite persuaded _all_ ownerReferences are wrong because autogenerating the UID is wrong in the first place, `kubeletconfig` is just the first failing here

Reply:
Yeah same, in this latest commit, I'm removing those and triggering an update on ownerref diff to allow performance profile to add them after the fact, from what I can see that doesn't seem to trigger a change on the machine config side. Testing now

Reply:
Per our conversation, ownership ref changes will be done in another PR